acceptance/test_helpers/with_separate_database.rb

```diff
@@ -0,0 +1,41 @@
+require "securerandom"
```

Comment:
nit: add a copyright header

You can copy one from any of the other files. Please make sure to update the year to 2023